README.md

```diff
@@ -62,12 +62,25 @@ For more information see:
  - For questions on the source code of ASPECT, portability, installation, etc., use the [aspect-devel](http://lists.geodynamics.org/cgi-bin/mailman/listinfo/aspect-devel) mailing list. This mailing list is where the ASPECT users and developers all hang out.
  - ASPECT is primarily based on the deal.II library. If you have particular questions about deal.II, contact the [deal.II discussion groups](https://www.dealii.org/mail.html).
  - In case of more general questions about mantle convection, you can contact the [CIG mantle convection mailing lists](http://lists.geodynamics.org/cgi-bin/mailman/listinfo/cig-MC).
- - If you have specific questions about ASPECT that are not suitable for public and archived mailing lists, you can contact the primary developers:
-
-    - Wolfgang Bangerth: bangerth@math.colostate.edu
-    - Timo Heister: heister@clemson.edu
-    - Rene Gassmoeller: rene.gassmoeller@mailbox.org
-    - Juliane Dannberg: judannberg@gmail.com
+ - ASPECT is being developed by a large, collaborative, and inclusive community. It is currently maintained by the following people:
+     - Wolfgang Bangerth: bangerth@math.colostate.edu
+     - Juliane Dannberg: heister@clemson.edu
+     - Rene Gassmoeller: rene.gassmoeller@mailbox.org
+     - Timo Heister: judannberg@gmail.com
```

Comment:
hrmmmm....

Reply:
hmm, you are right. I somehow managed to mess this up a bit ;)